cfgov/unprocessed/js/modules/transition/BaseTransition.js

```diff
@@ -138,6 +139,7 @@ function BaseTransition( element, classes ) {
   function _transitionComplete() {
     _removeEventListener();
     this.dispatchEvent( BaseTransition.END_EVENT, { target: this } );
+    _dom.classList.remove( BaseTransition.ANIMATING_CLASS );
```

Comment:
Swap this with line 141. (My mistake for saying it should go here. All cleanup for "animating" should happen before the end transition event fires.)

Reply:
Whoops, I think you mentioned this in chat and I forgot, thanks for calling it out again.